frontend/compositions/header/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react"
+import NPDCLogo from './assets/NPDCLogo.svg'
+import styles from './header.module.css'
+
+
+export default function Header() {
+  let [selected, setSelected] = useState('Search');
+
+  const handleNavChange = (e: any) => {
+    let value = e.target.innerText;
+    setSelected(value);
+  }
+
+  return (
+    <header className={styles.wrapper}>
+      <div className={styles.banner}>
+        <div className={styles.logo}>
+          <NPDCLogo />
+        </div>
+        <div className={styles.titleContainer}>
+          <h2 className={styles.title}>National Police Data Coalition</h2>
+          <p className={styles.subtitle}>The national index of police incidents</p>
+        </div>
+        <nav>
+          <ul className={styles.nav}>
+            <li className={selected === 'Search' ? styles.tab : ''} onClick={handleNavChange}>Search</li>
+            <li className={selected === 'Profile' ? styles.tab : ''} onClick={handleNavChange}>Profile</li>
+            <li className={selected === 'About' ? styles.tab : ''} onClick={handleNavChange}>About</li>
```

Comment:
We could factor out the classNames into a helper function to DRY this up a bit!

Reply:
Also! We'll need to make these a natively focusable element for it to be accessible to keyboard users

We'll probably want to review our routing strategy as a team, in the meantime adding anchor tags (inside of each `li` and around their text) as a placeholder is probably sufficient!

(in that case, we'll also want to move `handleNavChange` to the link click instead of the list item click)

Reply:
@srsexton94 thank you for all the feedback! I've added anchor tags with a "#" placeholder link for the meantime inside. of each `li` tag. 

I moved the full nav bar and the handleNavChange function into a separate component, and will factor out classNames into a helper function as well.

Reply:
hi @srsexton94 - i want to double check my understanding of making each `li`  natively focusable - my current understanding is that adding the anchor tags addressed this, i'm seeing that i'm able to tab through the links in sequential order. would it be best to also add a tabindex of 0 to all `li` elements, outside of the `a` tags?